apps/nextjs-website/src/lib/__tests__/header.test.ts

```diff
@@ -0,0 +1,139 @@
+import { describe, it, expect } from 'vitest';
+import { makeHeaderProps } from '../header'; // Adjust the import path based on your file structure
+import { Header } from '../fetch/header';
+import { CTAButtonType } from '../fetch/types/CTAButton';
+
+const parentNavItem = {
+  order: 1,
+  id: 1,
+  title: 'Parent',
+  path: 'parent',
+  parent: null,
+  menuAttached: true,
+};
+
+const childNavItem = {
+  order: 1,
+  id: 2,
+  title: 'Child',
+  path: 'child',
+  parent: parentNavItem,
+  menuAttached: true,
+};
```

Comment:
move these common structures in a file called `data.ts` under the `__tests__` folder. These structures are used also in other tests